src/EPR.Payment.Facade/Validations/RegistrationFees/RegulatorDtoValidator.cs

```diff
@@ -8,9 +9,8 @@ public class RegulatorDtoValidator : AbstractValidator<RegulatorDto>
         public RegulatorDtoValidator()
         {
             RuleFor(x => x.Regulator)
-                .NotEmpty().WithMessage("Regulator is required.")
```

Comment:
use CascadeMode.Stop

Reply:
they wanted to see all messages like <Regulator is required.; Invalid Regulator.>